test/unit/lib/DiscoveryStreamFeed.test.js

```diff
@@ -385,7 +385,7 @@ describe("DiscoveryStreamFeed", () => {
 
       await feed.loadSpocs(feed.store.dispatch);
 
-      assert.notCalled(global.fetch);
+      assert.calledOnce(global.fetch); // Persistent cache attempts a fetch
```

Comment:
This is kinda weird and might make test debugging confusing since the test description mentions `should not fetch`.
Can you instead stub out PersistentCache so that call doesn’t happen? It should prevent having to make all the other changes as well.

Reply:
yeah, good call